app/controllers/v0/message_drafts_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+module V0
+  class MessageDraftsController < SMController
+    def create
+      params = draft_create_params
+      draft = MessageDraft.new(params)
+      raise Common::Exceptions::ValidationErrors, draft unless draft.valid?
+      response = client.post_create_message_draft(params)
+
+      render json: response,
+             serializer: MessageSerializer,
+             meta:  {}
+    end
+
+    def update
+      params = draft_update_params
+      draft = MessageDraft.new(params)
+      raise Common::Exceptions::ValidationErrors, draft unless draft.valid?
+
+      response = client.post_create_message_draft(params)
+
+      render json: response,
+             serializer: MessageSerializer,
+             meta:  {}
```

Comment:
I think for updating a draft we don't really need to render the json. Just returning `head :no_content` should be sufficient. This is because the client side should know what the state is, including what the ID of the message is. Just telling them that the update is successful should be sufficient.